extensions/cometapi/src/api.ts

```diff
@@ -0,0 +1,45 @@
+import { getPreferenceValues } from "@raycast/api";
+import OpenAI from "openai";
+
+type Prefs = {
+  apikey: string;
+  model: string;
+};
```

Comment:
@greptileai, you didn't mention that the user shouldn't add the Prefs interface/type themselves since it's already autogenerated in `raycast-env.d.ts`